### PR TITLE
WFLY-15012 Fine/CoarseHotRodSessionExpirationTestCase fails intermittently

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-15012

Requires 
https://github.com/wildfly/wildfly/pull/14547

### CODE REVIEW

**Review by pferraro — APPROVED** [review]

**Comment by chengfang** [comment]
I guess the test class name is `CoarseHotRodSessionExpirationTestCase` in the title?

**Comment by rhusar** [comment]
@chengfang Of course! BTW Jira summary fields are not immutable, anyone with an account can just go ahead and fix it.

**Comment by pferraro** [comment]
Rebase on https://github.com/pferraro/wildfly/tree/WFLY-15088

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java [hunk 1]
```diff
@@ -434,43 +438,57 @@ public void test(@ArquillianResource(SessionOperationServlet.class) @OperateOnDe
             } finally {
                 HttpClientUtils.closeQuietly(response);
             }
+        }
 
-            // Trigger timeout of sessionId
-            Thread.sleep(AbstractClusteringTestCase.GRACE_TIME_TO_REPLICATE);
+        // Trigger timeout of sessionId - accounts for session timeout (1s) and infinispan reaper thread interval (1s)
+        // so that test conditions are typically met within the first attempt
+        Thread.sleep(TimeUnit.SECONDS.toMillis(2));
```
Comment (pferraro) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java:445]: We probably want to use TimeoutUtil.adjust(...) here.

**Review by pferraro — CHANGES_REQUESTED** [review]

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java (hunk 1 above)
Comment (rhusar) [reply testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java:445]: @pferraro I intentionally did not – the following retry logic handles any machines slowness.

Comment (pferraro) [reply testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/expiration/SessionExpirationTestCase.java:445]: That's true.

**Review by pferraro — APPROVED** [review]

**Pull request merged**